### DISCUSSION TITLE
Python client for Logto Management API

### DISCUSSION THREAD

**Discussion by ysemennikov** [logto-io/logto#6035] [Show and tell]
Hi everyone!

I'm excited to introduce a Python client I've developed for Logto: [py-logto-admin](https://github.com/ysemennikov/py-logto-admin).

## What It Does:

The [py-logto-admin](https://github.com/ysemennikov/py-logto-admin) client allows you to interact with the Logto Management API seamlessly, making it easier to integrate Logto's authentication and user management features into your Python applications.

## Key Features:

- Asynchronous Support: Fully supports asynchronous operations.
- Modern Python: Utilizes the latest Python features, including type annotations and dataclasses, for enhanced code clarity and robustness.
- Comprehensive Documentation: Includes thorough documentation and usage instructions directly within the docstrings.
- Built on top of the OpenAPI Spec: Built using the [openapi-python-client](https://github.com/openapi-generators/openapi-python-client), which means the client is generated from the [Logto OpenAPI Specification](https://openapi.logto.io/operation/operation-get-api-swagger-json).

I'd love for you to try it out and share your feedback!

**Comment by xiaoyijun** [comment]
Thank you for sharing this, and sorry for the late response!

Generating a client from the OpenAPI spec is exactly the approach we recommend for interacting with the Management API, so it is great to see a community package built this way. Nice work!

For reference, we now also publish [`@logto/api`](https://www.npmjs.com/package/@logto/api) for JavaScript and TypeScript, which follows the same idea of generating types and clients from the spec.

Thanks again for contributing to the community!
